src/main/scala/org/allenai/plugins/VersionInjectorPlugin.scala

```diff
@@ -117,6 +134,40 @@ object VersionInjectorPlugin extends AutoPlugin {
         gitConfFile
     }
 
+  val injectCacheKeyTask =
+    injectCacheKey <<= (resourceManaged in Compile, organization, name, version, streams, gitMostRecentCommit) map { // scalastyle: ignore
+      (resourceManaged, org, name, version, s, gitMRC) =>
+        val cacheKeyConfFile = resourceManaged / org / cleanArtifactName(name) /
+    }
+
+  val stageChecksumTask = stageChecksum := {
+    import java.io.{ File => JFile }
+
+    def getFileList(dir: String): Seq[JFile] = new JFile(dir) match {
+      case f if f.exists && f.isDirectory => f.listFiles.filter(_.isFile).toSeq
+      case _ => List[JFile]()
+    }
+
+    val stageDir = baseDirectory.value + "/target/universal/stage"
+    val allFiles: Seq[JFile] = DeployPlugin.autoImport.deployDirs.value
+      .map { dir => stageDir + "/" + dir }
+      .flatMap(getFileList)
+
+    val (filesToGetGitSha, filesToHash) = allFiles.partition {
```

Comment:
If you're using braces + spaces to invoke a method, you should invoke it infix style:
```scala
foo infix { args }
// versus
foo.standard(args)
```